### PR TITLE
Add mm:ss component

### PR DESCRIPTION
**Pull request by trevorgerhardt**
And switch Phasing minutes, Dwell time, and segment speeds to use it.

refs #320

### CODE REVIEW

**Comment by trevorgerhardt** [comment]
Allows time to be entered in with a decimal point also.

**Comment by trevorgerhardt** [comment]
@MarcovanderLinden @abyrd which inputs were you having trouble with? I can switch them to use this new component

**Comment by MarcovanderLinden** [comment]
At the moment I specifically have trouble entering frequency entries with decimal points (e.g. 7,5 minutes) this is in all modification types that ask for a frequency.

**Comment by abyrd** [comment]
@marcovanderlinden that's probably an internationalization issue. Please use the English decimal separator (a dot/full stop).

**Comment by MarcovanderLinden** [comment]
@abyrd , that's not it, i've tried 7,5 7.5 and 7:5, all of these show 75 or 0. I think that what @mattwigway said on slack makes sense: the system checks input really fast and doesn't recognize 7. and before i can type the 5 it turns the value to 0. 
His workaround works fine by the way (copying the number 7.5 in)

**Comment by trevorgerhardt** [comment]
@MarcovanderLinden this pull request should address that issue. We still need to switch the frequency input to use the new component and then deploy the fix to production.

File: lib/components/minutes-seconds.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  render () {
+    const {seconds, ...props} = this.props
+    return <Text
+      {...props}
+      onBlur={this._onBlur}
+      onChange={this._onChange}
+      onKeyDown={this._onKeyDown}
+      units='mm:ss'
```
Comment (mattwigway) [inline lib/components/minutes-seconds.js]: This needs to be a message so we can internationalize it.

File: lib/components/modification/segment-speeds.js [hunk 2]
```diff
@@ -172,12 +167,10 @@ export default class SegmentSpeeds extends DeepEqual {
                 value={speed}
                 units='km/h'
                 />
-              <Input
+              <MinutesSeconds
                 placeholder='Time'
```
Comment (mattwigway) [inline lib/components/modification/segment-speeds.js]: This should also be internationalized.

**Pull request merged**